include/kernel/gemm/default_gemm.hpp

```diff
@@ -151,26 +158,25 @@ struct default_gemm_selector_config_t
                                               wg_shape>,
                                       elem_v_t<tune_key::wg_tile_k, wg_tile_k>,
                                       elem_v_t<tune_key::gpu_arch,
-                                              gpu_arch_tag>>>>::type> {};
+                                              arch_tag>>>>::type> {};
 
 template <typename dtype_a, mem_layout mem_layout_a, uint32_t alignment_a,
         mem_space mem_space_a, typename dtype_b, mem_layout mem_layout_b,
         uint32_t alignment_b, mem_space mem_space_b, typename dtype_acc,
-        typename wg_shape, uint32_t wg_tile_k,
-        gpu_arch gpu_arch_tag = gpu_arch::Xe, typename tune_option = dict_t<>>
-struct default_gemm_selector_t
-    : default_gemm_selector_config_t<dtype_a, mem_layout_a, alignment_a,
-              mem_space_a, dtype_b, mem_layout_b, alignment_b, mem_space_b,
-              dtype_acc, wg_shape, wg_tile_k, gpu_arch_tag, tune_option>::type {
-};
+        typename wg_shape, uint32_t wg_tile_k, gpu_arch arch_tag = gpu_arch::Xe,
+        typename tune_option = dict_t<>>
+using default_gemm_selector_t = typename default_gemm_selector_config_t<dtype_a,
```

Comment:
Why did you use derivation before? This modification looks better.

Reply:
Yes, I preferred the templated type aliasing way. It is more consise, and won't be a barrier when debugging with `XETLA_PRINT<xxx_selector_t<...>>()`.